Dockerfile

```diff
@@ -1,34 +1,59 @@
 # https://docs.docker.com/reference/builder/
 # NOTE dockerignore is ignored https://github.com/docker/docker/issues/9455
 
-FROM ensime/ensime:v1.x
-MAINTAINER Sam Halliday, sam.halliday@gmail.com
+FROM debian:jessie
```

Comment:
actually, I'd be happy for you to add a line to the `ensime/ensime:latest` run that downloads coursier and runs the server update script to populate the caches. Do you want to just do that?